### ISSUE TITLE
Unable to get Firebase push notification  when Onesignal push notification is integrated in Android 

### ISSUE THREAD

**Issue by nishkul** [OneSignal/OneSignal-Android-SDK#646]
**Description:**
I am using firebase notification service in my android  aap and also integrated onesignal push notiifcation service in android , after integrating one signal in android unable to get firebase push notiifcation from dashboard . i dont understand what is the problem 

(your description here)

**Environment**

build.gradle file 

dependencies

 {

    implementation fileTree(dir: 'libs', include: ['*.jar'])

    implementation 'com.android.support:appcompat-v7:27.1.1'

    implementation 'com.android.support.constraint:constraint-layout:1.1.3'

    implementation 'com.android.support:design:27.1.1'

    testImplementation 'junit:junit:4.12'

    androidTestImplementation 'com.android.support.test:runner:1.0.2'

    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'

    implementation 'com.google.firebase:firebase-core:16.0.4'

    implementation 'com.google.firebase:firebase-messaging:17.3.3'

    def lifecycle_version = "1.1.1"

    implementation "android.arch.lifecycle:extensions:$lifecycle_version"

    annotationProcessor "android.arch.lifecycle:compiler:$lifecycle_version"

    implementation 'com.onesignal:OneSignal:[3.10.2, 3.99.99]'

}

**Comment by jkasten2** [comment]
@nishkul Thanks for the details. If you fully uninstalled and reinstalled the app on your device this will cause FCM to assign a new token to your device. Can you make sure you are sending to the new token?

Just to confirm, after adding the OneSignal SDK project notifications from the Firebase dashboard are no longer displaying but notifications sent from OneSignal are displaying?

Can you check the logcat for any errors or warnings with FCM or GCM in the message when you open the app or when you send a notification?

**Comment by nishkul** [comment]
Thanks for your response, yes I am sending a new device token whenever it's installed, I figure out the issue maybe it's because of firebase latest libary version is 
```
implementation 'com.google.firebase:firebase-messaging:17.3.3'
implementation 'com.google.firebase:firebase-core:16.0.2'
```
as I downgrade the library to 11 i.e 
```
implementation 'com.google.firebase:firebase-core:11.4.0'
 implementation 'com.google.firebase:firebase-messaging:11.4.0'
```
its working fine, I am clueless about this issue please help me what is wrong in these

**Comment by jkasten2** [comment]
@nishkul I wasn't able to see any conflicts between the following 3 dependencies.
```
implementation 'com.onesignal:OneSignal:3.10.2'
implementation 'com.google.firebase:firebase-messaging:17.3.3'
implementation 'com.google.firebase:firebase-core:16.0.4'
```
Since OneSignal is set to use older versions of `firebase-messaging` and the limitations on how Gradle handles interdependencies you will need to make sure you have the [OneSignal-Gradle-Plugin](https://github.com/OneSignal/OneSignal-Gradle-Plugin) setup in your project.

If you can double check for any FCM or GCM error message in your logcat when running into the issue it should help pin point the issue if you are still seeing a problem.

I am guessing you mean version `16.0.4` of `com.google.firebase:firebase-core`? Not `16.0.2`

**Comment by nishkul** [comment]
Okay  @jkasten2  I will try and let you know 
 yes I am using 

> 16.0.4

  version

**Comment by jkasten2** [comment]
We released OneSignal-Gradle-Plugin version 0.12.1 which I believe has fixed this issue. Please create an issue below if you are still seeing problem with all the details in it's template.
https://github.com/OneSignal/OneSignal-Gradle-Plugin/issues

Thanks.
